react-multiple-components/my-app/src/RotatingBanner.tsx

```diff
@@ -0,0 +1,112 @@
+import { useState } from 'react';
+
+type Props = {
+  items: string[];
+};
+type Banner = {
```

Comment:
I find the code easier to read if the props types are down with the function that uses them.